app/models/yard_sale.rb

```diff
@@ -0,0 +1,6 @@
+class YardSale < ApplicationRecord
+  belongs_to :user
+  has_many :images
+  has_one :address
+  has_many :yardsale_tags
```

Comment:
We don't need this is we're using act as taggable. Instead you can use acts_as_taggable_on :tags